packages/esm-appointments-app/src/tabs/appointments-table.component.tsx

```diff
@@ -23,36 +19,16 @@ import {
   TableToolbar,
   TableToolbarContent,
   TableToolbarSearch,
-  Tabs,
-  Tag,
   Tile,
-  TooltipDefinition,
 } from 'carbon-components-react';
 import Add16 from '@carbon/icons-react/es/add/16';
-import Group16 from '@carbon/icons-react/es/group/16';
-import InProgress16 from '@carbon/icons-react/es/in-progress/16';
-import { useLayoutType, ConfigurableLink } from '@openmrs/esm-framework';
-import {
-  useVisitQueueEntries,
-  useServices,
-  QueueService,
-  QueueStatus,
-  MappedVisitQueueEntry,
-  MappedQueuePriority,
-  useAppointmentEntries,
-} from './appointments-table.resource';
+import CurrencyPound16 from '@carbon/icons-react/es/currency--pound/16';
+import Omega16 from '@carbon/icons-react/es/omega/16';
```

Comment:
Are these the appropriate icons per the design or these are place holders for the time being?

Reply:
These are placeholders